src/LodeRunner.API/src/Controllers/LoadTestConfigsController.cs

```diff
@@ -91,11 +101,15 @@ public async Task<ActionResult> CreateLoadTestConfig([FromBody, SwaggerRequestBo
 
             if (newloadTestConfig.Validate(out string errorMessage, loadTestConfigPayload.PropertiesChanged))
             {
-                var insertedLoadTestConfig = await loadTestConfigService.Post(newloadTestConfig, cancellationTokenSource.Token);
+                var insertedLoadTestConfigResponse = await loadTestConfigService.Post(newloadTestConfig, cancellationTokenSource.Token);
 
-                if (insertedLoadTestConfig != null)
+                if (insertedLoadTestConfigResponse.Model != null && insertedLoadTestConfigResponse.StatusCode == HttpStatusCode.OK)
                 {
-                    return await ResultHandler.CreateResult(insertedLoadTestConfig, HttpStatusCode.OK);
+                    return await ResultHandler.CreateResult(insertedLoadTestConfigResponse.Model, HttpStatusCode.OK);
+                }
+                else if (insertedLoadTestConfigResponse.StatusCode == HttpStatusCode.BadRequest)
+                {
+                    return await ResultHandler.CreateBadRequestResult(insertedLoadTestConfigResponse.Errors, RequestLogger.GetPathAndQuerystring(this.Request));
```

Comment:
The final else hides the resulting object and return code.  The message should indicate the value of the return even if null and the value of the status code.

Consider changing the `SystemConstants.UnableToCreateLoadTestConfig` to have a string with placeholders such as:
```csharp
 public const string UnableToCreateTestRun = "Unable to create Test Run item.  The status code returned was {0} and the value returned was :{1}";
```

Then the final else could look like
```csharp
  return await ResultHandler.CreateErrorResult(string.FormatString(SystemConstants.UnableToCreateLoadTestConfig, insertedLoadTestConfigResponse.StatusCode, insertedLoadTestConfigResponse.Model ) , HttpStatusCode.InternalServerError);
```
That way we know what came back from CosmosDB if we're debugging.

Reply:
This will be addressed in a separate issue https://github.com/orgs/retaildevcrews/projects/12#card-76493062